src/requirements.txt

```
@@ -0,0 +1,6 @@
+numpy==1.23.3
+pandas==1.4.2
+plotly==5.4.0
+requests==2.28.1
+streamlit==1.20.0
+web3==5.25.0
```

```suggestion
web3==6.0.0
```
I think we should upgrade this library since some of these `camelCase` vs `snake_case` issues are resolved. I left comments throughout where a function can be renamed to make it consistent.